app/src/main/java/com/infomaniak/swisstransfer/ui/screen/newtransfer/upload/UploadProgressViewModel.kt

```diff
@@ -91,6 +91,10 @@ class UploadProgressViewModel @Inject constructor(
         }
     }
 
+    fun removeAllUploadSession() {
+        viewModelScope.launch(ioDispatcher) { uploadManager.removeAllUploadSession() }
+    }
```

Comment:
This works.
But I'm not certain it's the best place to do it.
Any insight is appreciated.